poseidon/export_test_vectors/src/vectors.rs

```diff
@@ -89,8 +90,8 @@ pub fn generate(mode: Mode, param_type: ParamType) -> TestVectors {
             .collect();
         let mut output_bytes = vec![];
         output
-            .into_repr()
-            .serialize(&mut output_bytes)
+            .0
```

Comment:
I would try to rely on a method on `output` instead of getting the inner element of the structure.
Also, simply adding a single test checking the serialization is still the same would be nice - it is used for further testing, not production code, but always good.